File: app/src/main/java/com/example/tipper/MainActivity.java
//Damian Eggert s19766
package com.example.tipper;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button goBmi, goCalories, goRecipes;

        goBmi = findViewById(R.id.bmiButton);
        goCalories = findViewById(R.id.caloriesButton);
        goRecipes = findViewById(R.id.recipesButton);

        goBmi.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                launchBmiCalculator();
            }
        });

        goCalories.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                launchCaloriesCalculator();
            }
        });

        goRecipes.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                launchGetReceip();
            }
        });
    }

    private void launchBmiCalculator() {
        Intent intent = new Intent(this, BmiActivity.class);
        startActivity(intent);
    }

    private void launchCaloriesCalculator() {
        Intent intent = new Intent(this, CaloriesCalcActivity.class);
        startActivity(intent);
    }

    private void launchGetReceip() {
        Intent intent = new Intent(this, RecipesActivity.class);
        startActivity(intent);
    }
}


/*************************************************************************
 * (C) Copyright 1992-2016 by Deitel & Associates, Inc. and               *
 * Pearson Education, Inc. All Rights Reserved.                           *
 *                                                                        *
 * DISCLAIMER: The authors and publisher of this book have used their     *
 * best efforts in preparing the book. These efforts include the          *
 * development, research, and testing of the theories and programs        *
 * to determine their effectiveness. The authors and publisher make       *
 * no warranty of any kind, expressed or implied, with regard to these    *
 * programs or to the documentation contained in these books. The authors *
 * and publisher shall not be liable in any event for incidental or       *
 * consequential damages in connection with, or arising out of, the       *
 * furnishing, performance, or use of these programs.                     *
 *************************************************************************/

File: app/src/main/java/com/example/tipper/MainActivity.kt
//Damian Eggert s19766
package com.example.tipper

import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.textfield.TextInputLayout
import android.widget.TextView
import android.os.Bundle
import com.example.tipper.R
import android.widget.EditText
import android.widget.Toast
import android.content.Intent
import com.example.tipper.BmiActivity
import com.example.tipper.CaloriesCalcActivity
import com.example.tipper.RecipesActivity
import com.example.tipper.QuizActivity
import com.example.tipper.Question
import android.annotation.SuppressLint
import android.widget.Button
import com.example.tipper.MainActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val goBmi: Button
        val goCalories: Button
        val goRecipes: Button
        val goQuiz: Button
        goBmi = findViewById(R.id.bmiButton)
        goCalories = findViewById(R.id.caloriesButton)
        goRecipes = findViewById(R.id.recipesButton)
        goQuiz = findViewById(R.id.goQuiz)
        goBmi.setOnClickListener { launchBmiCalculator() }
        goCalories.setOnClickListener { launchCaloriesCalculator() }
        goRecipes.setOnClickListener { launchGetReceip() }
        goQuiz.setOnClickListener { launchQuiz() }
    }

    private fun launchBmiCalculator() {
        val intent = Intent(this, BmiActivity::class.java)
        startActivity(intent)
    }

    private fun launchCaloriesCalculator() {
        val intent = Intent(this, CaloriesCalcActivity::class.java)
        startActivity(intent)
    }

    private fun launchGetReceip() {
        val intent = Intent(this, RecipesActivity::class.java)
        startActivity(intent)
    }

    private fun launchQuiz() {
        val intent = Intent(this, QuizActivity::class.java)
        startActivity(intent)
    }
}